.github/workflows/tests-cibw.yml

```
@@ -40,8 +44,45 @@ jobs:
 
     - run: brew upgrade cmake
 
-    - uses: pypa/cibuildwheel@v3.0
+    - uses: pypa/cibuildwheel@v3.1
       env:
         CIBW_PLATFORM: ios
+        CIBW_SKIP: cp314-*  # https://github.com/pypa/cibuildwheel/issues/2494
+      with:
+        package-dir: tests
+
+  build-android:
+    name: Android wheel ${{ matrix.runs-on }}
+    runs-on: ${{ matrix.runs-on }}
+    strategy:
+      fail-fast: false
+      matrix:
+        runs-on: [macos-latest, macos-13, ubuntu-latest]
+    steps:
+    - uses: actions/checkout@v4
+      with:
+        submodules: true
+        fetch-depth: 0
+
+    # GitHub Actions can't currently run the Android emulator on macOS.
+    - name: Skip Android tests on macOS
+      if: contains(matrix.runs-on, 'macos')
+      run: echo "CIBW_TEST_COMMAND=" >> "$GITHUB_ENV"
+
+    # https://github.blog/changelog/2024-04-02-github-actions-hardware-accelerated-android-virtualization-now-available/
+    - name: Enable KVM for Android emulator
+      if: contains(matrix.runs-on, 'ubuntu')
+      run: |
+        echo 'KERNEL=="kvm", GROUP="kvm", MODE="0666", OPTIONS+="static_node=kvm"' | sudo tee /etc/udev/rules.d/99-kvm4all.rules
+        sudo udevadm control --reload-rules
+        sudo udevadm trigger --name-match=kvm
+
+    # TODO: remove "wheel" once we're using a cibuildwheel version that includes
+    # https://github.com/pypa/cibuildwheel/pull/2515.
+    - run: pipx install wheel patchelf
```

```suggestion
    - run: pipx install patchelf
```

I guess we still need `patchelf`? Is that something cibuildwheel should install?